hooli_data_eng/resources/__init__.py

```diff
@@ -57,10 +58,23 @@ def get_env():
 # The dbt file dbt_project/config/profiles.yaml
 # specifies what databases to targets, and locally will
 # execute against a DuckDB
-
 DBT_PROJECT_DIR = file_relative_path(__file__, "../../dbt_project")
 DBT_PROFILES_DIR = file_relative_path(__file__, "../../dbt_project/config")
 
+dbt_project = DbtProject(
+    project_dir=DBT_PROJECT_DIR,
+    state_dir="target/slim_ci",
+    manifest_preparer=DagsterDbtManifestPreparer(
+        generate_cli_args=[
+            "--quiet",
+            "parse",
+            "--target",
+            get_env(),
+            "--profiles-dir",
+            DBT_PROFILES_DIR,
+        ],
+    ),
+)
```

Comment:
I can imagine that `target` could be a top level variable for `DbtProject`, so that users don't have to interface with `DagsterDbtManifestPreparer`.

Also assuming that `DBT_PROFILES_DIR` isn't in the `config/` directory (it's unnecessary.)

```suggestion
dbt_project = DbtProject(
    project_dir=DBT_PROJECT_DIR,
    state_dir="target/slim_ci",
    target=get_env(),
)
```

Reply:
changed to this, stacked on https://github.com/dagster-io/dagster/pull/21108

I get 
```
Error: Invalid value for '--profiles-dir': Path '/Users/alangenfeld/.dbt' does not exist.
```

so it seems like the profiles dir settings is needed

Reply:
Yeah we have to move `profiles.yml` to the top-level at `hooli-data-eng-pipelines/dbt_project/profiles.yml`. It's currently at `hooli-data-eng-pipelines/dbt_project/config/profiles.yml`.

Once we do that, we can read from it without explicitly specifying it.

Reply:
full error stack https://gist.github.com/alangenfeld/2b97ad0e352ea788ca7961efda5be1d1

Reply:
got it, its working now